openslides/motions/models.py

```diff
@@ -777,6 +507,42 @@ def get_paragraph_based_amendments(self):
         return list(filter(lambda amend: amend.is_paragraph_based_amendment(), self.amendments.all()))
 
 
+class MotionCommentField(RESTModelMixin, models.Model):
+    access_permissions = MotionCommentFieldAccessPermissions()
+
+    name = models.CharField(max_length=255)
+
+    read_groups = models.ManyToManyField(Group, blank=True, related_name='read_comments')
```

Comment:
Is it possible to avoid an import of "Group"? See how we refer to user in motion submitter and motion supporter.

Try `settings.AUTH_USER_MODEL.groups.field.related_model`

Reply:
The type of `settings.AUTH_USER_MODEL` is str, so no, this won't work.
Another possibility is `get_user_model().groups.field.related_model`, but then I get this error:
`django.core.exceptions.AppRegistryNotReady: Models aren't loaded yet.`
This is reasonable too, because `get_user_model()` just works, if the application is running.

So I'll stick to `Groups` until someone finds another solution

Reply:
I think the correct solution is to add a new settings variable AUTH_GROUP_MODEL which is a string `'user.Group'` or something like this.